maxim/tests/test_openai.py

```
@@ -105,107 +195,272 @@ def setUp(self):
-        # Cleanup the mock writer
-        self.mock_writer.cleanup()
+        print("Response 1: ", response1)
+        print("Response 2: ", response2)
+        self.logger.flush()
+
+    def tearDown(self) -> None:
         return super().tearDown()
+
+if __name__ == "__main__":
+    # Run async tests
+    async def run_async_tests():
```

Ruff (0.12.2)

Missing return type annotation for public function `run_async_tests`

Add return type annotation: `None`

(ANN201)